Haskell/Embeddings/HTML (for HSX).sublime-syntax

```diff
@@ -0,0 +1,47 @@
+%YAML 1.2
+---
+scope: text.html.embedded.haskell
+version: 2
+hidden: true
+
+extends: Packages/HTML/HTML.sublime-syntax
```

Comment:
Can you add a comment here or at the top of the file with a short summary for why we include HTML instead of XML here? I believe that would be a lot easier to find than the commit message.